pymilo/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """PyMilo modules."""
-from .pymilo_param import PYMILO_VERSION
-from .pymilo_obj import Export, Import
+from pymilo.pymilo_param import PYMILO_VERSION
```

Comment:
I also suggest removing this first `pymilo.` in importing sections.
It will lead to a smaller PR and also prevent future possible refactoring.

Reply:
applied. all Pymilo imports in the library files, are rolled back to the relative imports.